test/src/e2e/pay.test.ts

```diff
@@ -16,6 +16,7 @@
 
 import { expect } from "chai";
 import "mocha";
+import { aliceAddress, faucetAddress } from "../helper/constants";
```

Comment:
It seems `aliceAddress` is not used. Otherwise, LGTM.

Reply:
Thank you. I fixed it.